straxen/misc.py

```diff
@@ -79,3 +88,103 @@ def utilix_is_configured(header='RunDB', section='pymongo_database') -> bool:
                 straxen.uconfig.get(header, section) is not None)
     except NoSectionError:
         return False
+
+@export
+class TimeWidgets:
+
+    def __init__(self):
+        """Creates interactive time widgets which allow to convert a
+        human readble date and time into a start and endtime in unix time
+        nano-seconds.
+        """
+        self._created_widgets = False
+
+    def create_widgets(self):
+        """
+        Creates time and time zone widget for simpler time querying.
+
+        Note:
+            Please be aware that the correct format for the time field
+            is HH:MM.
+        """
+        utcend = datetime.utcnow()
+        deltat = timedelta(minutes=60)
+        utcstart = utcend - deltat
+
+        self._start_widget = self._create_date_and_time_widget(utcstart, 'Start')
+        self._end_widget = self._create_date_and_time_widget(utcend, 'End')
+        self._time_zone_widget = self._create_time_zone_widget()
+        self._created_widgets = True
+
+        return widgets.VBox([self._time_zone_widget,
+                             self._start_widget,
+                             self._end_widget])
+
+    def get_start_end(self):
```

Comment:
Did you not have a similar function in the scada interface, if so, can we reuse?

Reply:
In scada there is a function to convert the time index of a pandas dataframe into different time_zones. So unfortunately not.